pkg/finder/fsfinder.go

```diff
@@ -16,6 +16,7 @@ type FileSystemFinder struct {
 	FileTypes        []filetype.FileType
 	ExcludeDirs      []string
 	ExcludeFileTypes []string
+	Depth            *int
```

Comment:
@kehoecj This is now a pointer. That way if it's null/nil, we ignore it and behave same as before.